dev_guide/deployments.adoc

```diff
@@ -244,43 +260,50 @@ spec:
 
 == Viewing Deployment Logs
 
-To view the logs of the latest deployment for a given deployment configuration:
+To stream the logs of the latest revision for a given deployment configuration:
 
 ----
-$ oc logs dc/<deployment_config> [--follow]
+$ oc logs -f dc/<name>
```

Comment:
@fabianofranz @smarterclayton is there a convention about using flags before vs after the resource?